language/de-DE/mod_articles.ini

```
@@ -26,7 +26,8 @@ MOD_ARTICLES_FIELD_DATEGROUPINGFIELD_DESC="Auswählen, welches Datumsfeld zur Gr
 MOD_ARTICLES_FIELD_DATEGROUPINGFIELD_LABEL="Datumsgruppierungsfeld"
 MOD_ARTICLES_FIELD_DATERANGEFIELD_LABEL="Datumsbereich"
 MOD_ARTICLES_FIELD_ENDDATE_LABEL="bis Datum"
-MOD_ARTICLES_FIELD_EXCLUDEDARTICLES_LABEL="Beitrags-IDs ausschließen"
+MOD_ARTICLES_FIELD_EX_OR_INCLUDE_LABEL="Beitrag ein- oder ausschließen"
```

```suggestion
MOD_ARTICLES_FIELD_EX_OR_INCLUDE_LABEL="Beiträge ein- oder ausschließen"
```
Ist doch Mehrzahl hier oder?